app/upgrades.go

```diff
@@ -1,78 +1,24 @@
 package app
 
 import (
-	"fmt"
-
-	sdkmath "cosmossdk.io/math"
-	storetypes "github.com/cosmos/cosmos-sdk/store/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/types/module"
 	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
-	ibcfeetypes "github.com/cosmos/ibc-go/v5/modules/apps/29-fee/types"
 )
 
-func (app *App) RegisterUpgradeHandlers(experimental bool) {
-	upgradeHandlerV1 := func(ctx sdk.Context, _ upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
+func (app *App) RegisterUpgradeHandlers() {
+	upgradeHandlerV2 := func(ctx sdk.Context, _ upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
 		m, err := app.mm.RunMigrations(ctx, app.configurator, fromVM)
 		if err != nil {
 			return m, err
 		}
-		// clear extra_eips from evm parameters
-		// Ref: https://github.com/crypto-org-chain/cronos/issues/755
-		params := app.EvmKeeper.GetParams(ctx)
-		params.ExtraEIPs = []int64{}
-
-		// fix the incorrect value on testnet parameters
-		zero := sdkmath.ZeroInt()
-		params.ChainConfig.LondonBlock = &zero
-
-		app.EvmKeeper.SetParams(ctx, params)
+		gravParams := app.GravityKeeper.GetParams(ctx)
+		gravParams.GravityId = "cronos_gravity_testnet"
```

Comment:
Should we remove the `_testne` suffix or make the upgrade plan name to `v2.0.0-testnet3`?

Reply:
reverted the change in the upgrade handler because its breaking the integration test for some reason. Will do it in a separate PR